libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -2,11 +2,13 @@
 $month-navigator-width: 80px;
 
 table.calendar-table {
-  width: 100%;
+  width: calc(100% - 2 * #{size('xxxs')});
   border-collapse: collapse;
   user-select: none;
   outline: none !important;
   border-spacing: 0;
+  margin: size('xxxs');
+  margin-top: 0;
 
   thead {
     padding: 0.5em;
```

Comment:
This has no effect. Padding should be applied to the table cells (`<td>` or `<th>`).

I'm aware you didn't add this, but I think it's worth cleaning up